src/Elcodi/Component/EntityTranslator/Form/Type/TranslatableFieldType.php

```diff
@@ -168,10 +168,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 : '';
 
             $builder->add($translatedFieldName, $fieldType, array(
-                'required' => $this->evaluateRequired(
-                    $fieldOptions['required'],
-                    $locale
-                ),
+                'required' => $fieldOptions['required']
```

Comment:
if this is not always `bool`, maybe better check with `isset`. If it is, `evaluateRequired` will always receive `true` as first parameter.